cypress/fixtures/fireboltCalls/usergrants.json

```diff
@@ -100,5 +100,463 @@
     "SEND_ERROR_FROM_CHALLENGE_PROVIDER":{
         "method": "acknowledgeChallenge.challengeError",
         "params": "USERGRANTS_CHALLENGEERROR_PARAMS"
+    },
+    "GET_USERGRANTS_FOR_3RD_PARTY_APP": {
+        "method": "manage_usergrants.app",
+		"params": "USERGRANTS_3RD_PARTY_APP"
+    },
+    "EXPECTED_LIST_OF_GRANTS_FOR_3RD_PARTY_APP": {
+        "method": "usergrants.app",
+        "validationJsonPath": "result",
+        "content": "USERGRANTS_APP_DEVICE_ID_GRANTED_ROLE_USE"
+    },
+    "GET_USERGRANTS_FOR_DEVICE": {
+        "method": "manage_usergrants.device"
+    },
+    "EXPECTED_LIST_OF_GRANTS_FOR_DEVICE": {
+        "method": "usergrants.device",
+        "validationJsonPath": "result",
+        "content": "USERGRANTS_DEVICE_DISTRIBUTOR_GRANTED_ROLE_USE"
+    },
+    "GET_USERGRANTS_FOR_DEVICE_ID_CAPABILITY": {
+        "method": "manage_usergrants.capability",
+		"params": "USERGRANTS_DEVICE_ID"
+    },
+    "EXPECTED_LIST_OF_GRANTS_FOR_DEVICE_ID_CAPABILITY": {
+        "method": "usergrants.capability",
+        "validationJsonPath": "result",
+        "content": "USERGRANTS_CAPABILITY_DEVICE_ID_GRANTED_ROLE_USE"
+    },
```

Comment:
why is line 122 having manage, but not line 126 . Both are calling same method right

Reply:
same comment for 131 and 135 . Check for others too

Reply:
manage this is required only to make a call. I don't think there's a need to include "manage" in the validation.